server/budgettracker.business/Api/BudgetApi.cs

```diff
@@ -57,7 +56,7 @@ public async Task<ApiResponse> CreateBudget(ApiRequest request)
                 return new ApiResponse(ex.Message);
             }
 
-            CreateBudgetResponseContract response = _budgetConverter.ToResponseContract(newBudget);
+            BudgetResponseContract response = BudgetApiConverter.ToResponseContract(newBudget);
```

Comment:
So you want to only return on budget response contact for all the budget endpoints?

Reply:
This is a general use case class. Most api endpoints that return budget models will return the same fields. So lets have them all return the same type then. If one API endpoint returns a different format, then we can always write one off classes but for consistency sake, let's keep them as consistent as possible just like we do with the business model or data model. They are all maybe used differently but they all have the same fields within their respective layers.

Reply:
completely agree